src/SOS/SOS.Extensions/ModuleServiceFromDebuggerServices.cs

```diff
@@ -34,8 +81,8 @@ class ModuleFromDebuggerServices : Module, IModuleSymbols
                 ulong imageBase,
                 ulong imageSize,
                 uint indexFileSize,
-                uint indexTimeStamp) 
-                    : base(moduleService.Target)
+                uint indexTimeStamp)
+                : base(moduleService.Services)
```

Comment:
Targets, Modules, Runtimes and Threads or any thing derived are created directly by the appropriate service (ModuleService for Modules, etc.) so it doesn't go though the DI/Utilities.CreateInstance. ImportService attr's are not honored.

This specific change is because these classes can be instanced from a cloned target service container.  So none of these classes should use Target.Services directly and should use the IServiceProvider instance passed in the service factory.  Take a look at TargetFromDebuggerServices's constructor where the IModuleService factory is added and follow that done to where the Module/ModuleFromDebuggerService instance is created.